sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/core/ReactiveCosmosTemplateIT.java

```diff
@@ -281,6 +308,35 @@ public void testUpsert() {
         assertThat(responseDiagnosticsTestUtils.getCosmosDiagnostics()).isNotNull();
     }
 
+    @Test
+    public void testPatch() {
+        final Mono<Person> patch = cosmosTemplate.patch(insertedPerson, operations, null);
+        StepVerifier.create(patch).expectNextCount(1).verifyComplete();
+        Mono<Person> patchedPerson = cosmosTemplate.findById(containerName, insertedPerson.getId(), Person.class);
+        StepVerifier.create(patchedPerson).expectNextCount(1).verifyComplete();
+        assertEquals(patchedPerson.block().getAge(), PATCH_AGE_1);
```

Comment:
wonder whether the age verification can also be part of StepVerifier flow -> `expectNextMatches(person -> person.getAge() == PATCH_AGE_1)`

Reply:
added